substrate/client/transaction-pool/src/graph/validated_pool.rs

```diff
@@ -747,12 +747,20 @@ impl<B: ChainApi> ValidatedPool<B> {
 		self.listener.write().retracted(block_hash)
 	}
 
+	/// Refer to [`Listener::create_dropped_by_limits_stream`] for details.
 	pub fn create_dropped_by_limits_stream(
 		&self,
-	) -> super::listener::DroppedByLimitsStream<ExtrinsicHash<B>, BlockHash<B>> {
+	) -> super::listener::AggregatedStream<ExtrinsicHash<B>, BlockHash<B>> {
 		self.listener.write().create_dropped_by_limits_stream()
 	}
 
+	/// Refer to [`Listener::create_aggregated_stream`]
+	pub fn create_aggregated_stream(
```

Comment:
In general these changes mean that the `submit_and_watch` codepath is only used by the single state pool, so we can remove all of that once the fatxpool is the default.